docs/reference/api/widgets/button.rst

```diff
@@ -29,14 +29,8 @@ A button has a text label. A handler can be associated with button press events.
 
     button = toga.Button('Click me', on_press=my_callback)
 
-Notes
------
-* Any "truthy" object (i.e, object that evaluates as ``True``) can be provided
-  as the text for the button; if the object is not a string, it will be
-  automatically converted to a string using ``str()``.
-
-* The button text cannot contain newlines. Any content after the first newline
-  will be ignored.
```

Comment:
I've moved the property documentation into the docstring so it'll appear next to the property itself, rather than in a separate section.